### PR TITLE
Shadowlands/TheaterOfPain: Enable Trash module

### PR DESCRIPTION
**Pull request by mbelsky**
Hey,

There are a few changes that enable ToP Trash module and add some alerts.

### CODE REVIEW

File: Shadowlands/TheaterOfPain/Trash.lua [hunk 1]
```diff
@@ -3,38 +3,93 @@
...
+	self:Log("SPELL_CAST_START", "BoneSpear", 342675)
 end
 
 --------------------------------------------------------------------------------
 -- Event Handlers
 --
+
+-- Ancient Captain
+function mod:DemoralizingShout(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
```
Comment (Zireael-N) [inline Shadowlands/TheaterOfPain/Trash.lua:69]: This spell looks important enough to use `"red"` as a colour and `self:Interrupter() and "warning" or "alert"` as a sound.

File: Shadowlands/TheaterOfPain/Trash.lua [hunk 2]
```diff
@@ -3,38 +3,93 @@
...
+
+-- Rancid Gasbag
+function mod:VileEruption(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+end
+
+-- Maniacal Soulbinder
+function mod:NecroticBoltVolley(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
```
Comment (Zireael-N) [inline Shadowlands/TheaterOfPain/Trash.lua:87]: Same as Demoralizing Shout.

File: Shadowlands/TheaterOfPain/Trash.lua [hunk 3]
```diff
@@ -3,38 +3,93 @@
...
+
+-- Blighted Sludge-Spewer
+function mod:WitheringDischarge(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+end
+
+-- Rancid Gasbag
+function mod:VileEruption(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
```
Comment (Zireael-N) [inline Shadowlands/TheaterOfPain/Trash.lua:81]: I'd change the colour to `"yellow"`, it's an ability that's very easy to avoid.

**Review by Zireael-N — CHANGES_REQUESTED** [review]
Thanks for your pull request.

Please review your colour + sound usage, you used "orange" and "alert" everywhere, but it's meant to be used to tell the player how urgent it is to react to a warning. And you also want to avoid having a similar combo for a different spell coming from the same mob or mobs nearby, if possible.

**Comment by mbelsky** [comment]
> Please review your colour + sound usage, you used "orange" and "alert" everywhere, but it's meant to be used to tell the player how urgent it is to react to a warning. And you also want to avoid having a similar combo for a different spell coming from the same mob or mobs nearby, if possible.

@Zireael-N thank you for review and the explanation. Requested changes has been applied. 

Just first time contributing in BW/LW so I don't learn best combos yet for color and type of sound. Is there a more detailed guide or doc that describes color and sound choosing?

**Comment by funkydude** [comment]
> Is there a more detailed guide or doc that describes color and sound choosing?

Nope, just experience over time :)

**Pull request merged**